teuthology/openstack/__init__.py

```diff
@@ -391,24 +389,27 @@ def cloud_init_wait(self):
                             continue
                     log.exception('cloud_init_wait ' + ip)
                     raise
-                log.debug('cloud_init_wait ' + all_done)
+                log.debug('cloud_init_wait ' + tail)
                 try:
-                    stdin, stdout, stderr = client.exec_command(all_done)
-                    stdout.channel.settimeout(5)
-                    out = stdout.read()
-                    log.debug('cloud_init_wait stdout ' + all_done + ' ' + out)
+                    transport = client.get_transport()
```

Comment:
it's  a series of call to get access to the I/O channel which can be used to display things line by line